looprpc/client.proto

```diff
@@ -319,6 +341,23 @@ enum SwapState {
     INVOICE_SETTLED = 5;
 }
 
+message GetSwapsRequest {
+}
+
+message GetSwapsResponse {
+    /**
+    The list of all currently known swaps and their status.
+    */
+    repeated SwapStatus swaps = 1;
+}
+
+message GetSwapRequest {
+    /**
+    The swap identifier which currently is the hash that locks the HTLCs.
+    */
+    string id = 1;
```

Comment:
We should accept raw bytes here (the payment hash) rather than a string here. I get it makes it easier to use for REST (just pass the hex encoded version), but if we use our proto buf struct, then we'll ensure we display the payment hashes as hex all throughout.

Reply:
I chose `string` because in the existing messages (`SwapResponse` and `SwapStatus`) the id field is already a hex encoded `string` field. I think we should stay consistent in the RPC and either use string for the new messages as well or switch everything to raw bytes (and break existing gRPC users).
I think base64 is quite ugly, as it's also not URL safe. So there's an additional step required for a REST user: `hex -> raw bytes -> base64 -> encode URI component`.

What I think we really should do for `lnd v0.10.0` (and consequentially for loop too) is to patch the `grpc-gateway` library to get rid of base64 completely.

Reply:
Discussed offline that we should deprecate the `string id` and go for a `bytes` version instead.

Reply:
Shouldn't this be `bytes` now then?

Reply:
Yes, of course. Fixed.